examples/CMakeLists.txt

```diff
@@ -17,8 +17,9 @@ set(ENV{PKG_CONFIG_PATH} "${PKG_CONFIG_CHANGED_PATH}")
 
 find_package(LibtorrentRasterbar REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
+find_package(OpenSSL REQUIRED)
```

Comment:
if libtorrent is built without libtorrent support, will OPENSSL_INCLUDE_DIR still be defined? or will it just be an empty string and not cause any problems here?